content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -122,33 +122,30 @@ By default, Kubernetes uses the
 {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
 to interface with your chosen container runtime.
 
-kubeadm automatically tries to detect the container runtime
-by scanning through a list of well known Unix domain sockets. The detectable
-runtimes and the socket paths that are tried are:
-
-| Runtime    | Domain Socket                    |
-|------------|----------------------------------|
-| Docker     | /var/run/docker.sock             |
-| containerd | /run/containerd/containerd.sock  |
-| CRI-O      | /var/run/crio/crio.sock          |
+If you don't specify a runtime, kubeadm automatically tries to detect an installed
+container runtime by scanning through a list of well known Unix domain sockets.
+The following table lists container runtimes and their associated socket paths:
+
+{{< table caption = "Container runtimes and their socket paths" >}}
+| Runtime    | Path to Unix domain socket        |
+|------------|-----------------------------------|
+| Docker     | `/var/run/docker.sock`            |
+| containerd | `/run/containerd/containerd.sock` |
+| CRI-O      | `/var/run/crio/crio.sock`         |
+{{< /table >}}
 
 <br />
-If both Docker and containerd are detected together, Docker takes precedence. This is
-needed, because Docker 18.09 ships with containerd and both are detectable.
-If any other two or more runtimes are detected, kubeadm will exit with an appropriate
-error message.
+If both Docker and containerd are detected, Docker takes precedence. This is
+needed because Docker 18.09 ships with containerd and both are detectable even if you only
+installed Docker.
+If any other two or more runtimes are detected, kubeadm exits with an error.
```

Comment:
@sftim , `nit`, line 141, what does this phrase mean, `If any other two or more runtimes`? Should line 141 start with:
If more than two container runtimes are detected, `kubeadm` exits ...

Reply:
If `kubeadm` detects two or more runtimes, then there is special case handling if the two runtimes are containerd and Docker. Otherwise, kubeadm exits with an error.

I'll see about rewording that; it was a little confusing in the original page too, I think.

Might move the link inside the tabs.